File: _01_Languages/Kotlin/src/_03_clases/_01_ClaseYObjeto.kt
/*
 * 🧩 Enunciado:
 * Crea una clase `Persona` con atributos `nombre` y `edad`.
 * Instancia un objeto y muestra sus datos.
 */

class Persona(val nombre: String, val edad: Int) {
    fun mostrarDatos() {
        println("Nombre: $nombre, Edad: $edad")
    }
}

fun main() {
    val persona = Persona("Carlos", 25)
    persona.mostrarDatos()
}

File: _01_Languages/CSharp/src/_02_poo/_01_ClaseYObjeto.cs
/*
 * 🧩 Enunciado:
 * Crea una clase `Persona` con los atributos `nombre` y `edad`.
 * Instancia un objeto y muestra sus datos por consola.
 */

using System;

class Persona
{
    public string Nombre;
    public int Edad;

    public void MostrarDatos()
    {
        Console.WriteLine($"Nombre: {Nombre}, Edad: {Edad}");
    }
}

class _01_ClaseYObjeto
{
    static void Main()
    {
        Persona p1 = new Persona();
        p1.Nombre = "Lucía";
        p1.Edad = 22;

        p1.MostrarDatos();
    }
}
